src/mach_o.rs

```diff
@@ -372,6 +375,9 @@ impl MachO {
                 LoadCommand::DyldInfo { .. } => {
                     log!("Warning! DyldInfo is not handled.");
                 }
+                LoadCommand::UnixThread { state, .. } => {
```

Comment:
Hmm, it looks like there's also `LoadCommand::EntryPoint`, but I haven't checked if iPhone OS/iOS apps use this or when they started doing so.

Reply:
Another thing I'm wondering about is whether there can be several of these load commands? Does the app always have exactly one initial thread? I think the safest thing would be to do something like:

```rust
// Assumption: there is only one initial thread
assert!(thread_state.is_none());
thread_state = Some(state);
```

Reply:
`LoadCommand::EntryPoint` is the new version of `LoadCommand::UnixThread`. Seems like Apple switched to it at the same time as they went 64-bit.

I believe it's only supposed to be one. (https://opensource.apple.com/source/xnu/xnu-2050.18.24/EXTERNAL_HEADERS/mach-o/loader.h)
> For executable objects that are unix processes there will be one thread_command (cmd == LC_UNIXTHREAD) created for it by the link-editor. This is the same as a LC_THREAD, except that a stack is automatically created (based on the shell's limit for the stack size).  Command arguments and environment variables are copied onto that stack.

Reply:
Ah, if it's a 64-bit thing then I guess we don't have to worry about it.